source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.md

```diff
@@ -633,6 +751,11 @@ if connection state is "pending":
     decrement pendingConnectionCount
 else:
     decrement availableConnectionCount
+
+error = await_pending_response(pool, connection)
+if error:
+  return error
```

Comment:
The pseudocode that existed before the current PR used `throw`, for example, `throw PoolClosedError`. The new pseudocode should continue using that, instead of coming up with new syntax.

What is worse, is that the new pseudocode uses `return` here, but `raise` is two different places. That is, it is not even consistent within itself.